modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductDetails.testcase

```diff
@@ -1618,6 +1618,81 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-10897. Users cannot view XSS with Availability Estimates in product details"
+	@priority = 3
+	test CannotExecuteXSSWithAvailabilityEstimatesInProductDetails {
+		property portal.acceptance = "false";
+
+		task ("Given a Minium site") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+		}
+
+		task ("And a regular user and Minium site membership is created") {
+			JSONUser.addUser(
+				userEmailAddress = "testuser@liferay.com",
+				userFirstName = "Test",
+				userLastName = "User",
+				userScreenName = "testuser");
+
+			JSONUser.setFirstPassword(
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "testuser@liferay.com");
+
+			JSONUser.addUserToSite(
+				groupName = "Minium",
+				userEmailAddress = "testuser@liferay.com");
+		}
+
+		task ("And a product 'ABS Sensor' with Availability Estimates XSS entry is updated") {
```

Comment:
"And a new Availability Estimate is created using an XSS and the product 'ABS Sensor' is updated with this entry"